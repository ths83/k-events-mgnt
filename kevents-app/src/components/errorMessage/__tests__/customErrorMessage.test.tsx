import React from "react";
import renderer from "react-test-renderer";
import ErrorMessage from "../CustomErrorMessage";

test("renders component", () => {
  const item = renderer.create(<ErrorMessage message="" />).toJSON();
  expect(item).toMatchSnapshot();
});
