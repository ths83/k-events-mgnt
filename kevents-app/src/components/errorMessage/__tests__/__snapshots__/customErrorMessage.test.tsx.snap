// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
<h5
  className="ant-typography ant-typography-danger message"
  id="errorMessage"
  onClick={[Function]}
  style={
    Object {
      "WebkitLineClamp": undefined,
    }
  }
>
  
</h5>
`;
